tests/testsuite/runconfiguration.go

```diff
@@ -23,5 +23,20 @@ func initRunConfiguration(virtClient kubecli.KubevirtClient) {
 		runConfig.WarningToIgnoreList = append(runConfig.WarningToIgnoreList, "EvictionStrategy is set but vmi is not migratable")
 	}
 
+	if !hasRegisteredSlirpBindingPluginImage(kv) {
```

Comment:
Instead of ignoring it, I would add to the relevant tests an expect that expects getting the warning.

Reply:
Done